clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/validator/KubernetesValidationUtil.java

```diff
@@ -93,6 +100,25 @@ public boolean validateV2Credentials(AccountCredentialsProvider provider, String
       return false;
     }
 
+    if (!validateNamespace(namespace, (KubernetesV2Credentials)credentials.getCredentials())) {
+      return false;
+    }
+
+    return true;
+  }
+
+  protected boolean validateNamespace(String namespace, KubernetesV2Credentials credentials) {
+    final List<String> configuredNamespaces = credentials.getNamespaces();
+    if (configuredNamespaces != null && !configuredNamespaces.isEmpty() && !configuredNamespaces.contains(namespace)) {
+      reject( "wrongNamespace", "namespace");
```

Comment:
Extra space before `"`

Reply:
Also, should the string `"namespace`" be replaced with the actual requested namespace?